content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/125-introspection.mdx

```diff
@@ -93,11 +83,6 @@ model User {
 To be able to join data using the Prisma Client, you can add the [`@relation`](/reference/api-reference/prisma-schema-reference#relation) attributes to our models:
 
 ```prisma file=prisma/schema.prisma highlight=15;add|21;add
-generator client {
```

Comment:
Same as https://github.com/prisma/docs/pull/3037/files#r842155051

Removing any line here also means the `highlight` values need to be adapted.

Reply:
Done